packages/neuron-wallet/src/services/settings.ts

```diff
@@ -54,6 +62,9 @@ export default class SettingsService extends Store {
     if (!this.ckbDataPath) {
       this.ckbDataPath = path.resolve(app.getPath('userData'), 'chains/mainnet')
     }
+    if (!this.lightDataPath) {
+      this.lightDataPath = path.resolve(app.getPath('userData'), 'chains/light')
```

Comment:
Neuron only supports mainnet full node, so the path could be `chains/mainnet`. Now light client will be supported for mainnet and testnet, maybe `chains/light/testnet` is more clear in this case

Reply:
Good idea, and I think we need to rename `lightDataPath ` to `testnetLightDataPath` because I think we will add a feature to set it as editable. So we can edit `testnetLightDataPath` and `mainnetLightDataPath` separately.